### PR TITLE
Fix nightly tests by finding a case which has iterations and surfaces

### PR DESCRIPTION
**Pull request by roywilly**
## Issue
_Link to the issue resolved by this PR._

## Short description
_Short description of the approach taken to solve the issue_

## Pre-review checklist
- [ ] Read through all changes. No redundant `print()` statements, commented-out code, or other remnants from the development. 👀
- [ ] Make sure tests pass after every commit. ✅
- [ ] New/refactored code is following same conventions as the rest of the code base. 🧬
- [ ] New/refactored code is tested. ⚙
- [ ] Documentation has been updated 🧾

## Pre-merge checklist
- [ ] Commit history is consistent and clean. 👌

### CODE REVIEW

File: tests/test_access/tst_access_drogon_manage_login.py [hunk 1]
```diff
@@ -101,7 +101,10 @@ def test_aggregations_bulk(explorer: Explorer):
     cases = explorer.cases
     print("Number of cases: ", len(cases))
     assert len(cases) > 0
-    case = cases[0]
+    for c in cases:
+        if len(c.iterations) > 0 and len(c.surfaces) > 40:
+            case = c
+            break
```
Comment (equinor-ruaj) [inline tests/test_access/tst_access_drogon_manage_login.py:107]: @adnejacobsen @rwiker 
Is the Explorer able to do this filtering for us?

File: tests/test_access/tst_access_drogon_manage_login.py [hunk 2]
```diff
@@ -121,10 +124,13 @@ def test_aggregations_fast(explorer: Explorer):
...
     case_uuid = case.metadata.get("fmu").get("case").get("uuid")
```
Comment (equinor-ruaj) [inline tests/test_access/tst_access_drogon_manage_login.py:131]: If there are no valid cases, so when `case=None`, will we get a useful error message?

File: tests/test_access/tst_access_drogon_manage_login.py (hunk 1 above)
Comment (roywilly) [reply tests/test_access/tst_access_drogon_manage_login.py:108]: I assume no answer means no such support

**Review by equinor-ruaj — APPROVED** [review]
lgtm

**Pull request merged**